problems/balanced-brackets/balanced-brackets.go

```diff
@@ -36,6 +36,11 @@ func balanced_brackets(s string) bool{
 			}
 		}
 	}
+    // if remaining queue is not empty. ex: [[(())
+    if len(queue) != 0 {
```

Comment:
This could be further simplified to, no?

```suggestion
    return len(queue) == 0
```

Reply:
absolutely, you are right.